packages/lesswrong/server/recommendations/FeatureStrategy.ts

```diff
@@ -32,6 +32,9 @@ class FeatureStrategy extends RecommendationStrategy {
     let args = {};
 
     for (const {feature: featureName, weight} of features) {
+      if (weight === 0) {
```

Comment:
What's this for?

Reply:
Each part of the score is a particular expression multiplied by a weight, and these are then summed together. This is just a small optimisation to avoid calculating a particular part of the score if the weight multiplier is 0, since it won't matter anyway.